src/Stan/Config.hs

```diff
@@ -242,20 +242,21 @@ Haskell project and outputs all 'Observation's it finds. Using
 The algorithm for figuring out the resulting set of 'Inspection's per
 module applies each 'Check' one-by-one in order of their appearance.
 
-When introducing a new 'Check' in the config, you must always specify
-the 'CheckType' (either 'Include' or 'Ignore'). 'CheckFilter' and
-'CheckScope' can be omitted. If they are not written explicitly, then
-the 'Check' is applied to all the entries. Specifically:
-
-* If 'CheckFilter' is not specified, 'Check' applies to all
-  inspections for the given 'CheckScope' (either 'Ignore' or 'Include'
-  all 'Inspection's)
-* If 'CheckScope' is not specified, 'Check' applies given
-  'CheckFilter' to all files
-
-As a result of this approach, when both 'CheckFilter' and 'CheckScope'
-are not specified explicitly, either all 'Inspection's are 'Ignore'd,
-or all inspections are 'Include'd back.
+When introducing a new 'Check' in the config, you must always three
+key-value pairs:
+
+1. 'CheckType' — control inclusion and exclusion criteria
+    * 'Include'
+    * 'Ignore'
+2. 'CheckFilter' — how to filter inspections
+    * 'CheckInspection': by specific 'Inspection' 'Id'
+    * 'CheckSeverity': by specific 'Severity'
+    * 'CheckCategory': by specific 'Category'
+    * 'CheckAll': applied to all 'Inspection's
+3. 'Scope' — where to apply check
+    * 'ScopeFile': only to the specific file
+    * 'ScopeDirectory': to all files in the specified directory
+    * 'ScopeAll': to all files
 
 The algorithm doesn't remove any files or inspections from the
 consideration completely. So, for example, if you ignore all
```

Comment:
Could you please add a comment that this could be done with the `remove` fields now? :pray:

Reply:
Thank you :heartbeat: